lib/request.js

```diff
@@ -7,14 +7,13 @@
 /*!
  * Module dependancies
  */
+import superagent from "superagent"
+import http from "http"
+import https from "https"
+import methods from "methods"
+import util from "util"
 
-var http = require('http')
-  , https = require('https')
-  , methods = require('methods')
-  , superagent = require('superagent')
-  , Agent = superagent.agent
-  , Request = superagent.Request
-  , util = require('util');
+var Agent = superagent.agent, Request = superagent.Request;
```

Comment:
can you make these individual `const` declarations?

we should probably add lint rules and a formatter in a separate PR to do this throughout the repo

Reply:
Yes, prettier and eslint would be pretty nice, I could add them in the future

Another way to write this could be 
```js
const { Agent, Request } = superagent;
```

or rename it in the import statement
```js
import {agent as Agent, Request} from "superagent"
```

maybe these are even better, but I'm not shure